.settings/language.settings.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What is this file used for? Is it for saving cross compiler settings when using PPSDK?

Reply:
yes, it is